react/.eslintignore

```diff
@@ -1,5 +1,8 @@
-tests/
+backstop/
 es/
 lib/
+node_modules/
+storybook-static/
+tests/
```

Comment:
We eslintignore tests on search as well, but I'm not sure if it's something we should be considering changing long term. It definitely had led to things slipping through the cracks (unused variables & imports, hanging spaces, etc). Thoughts?

Reply:
I agree we should probably lint tests, too -- I'll note in the ticket for followup.

Reply:
Also note: this is for the test that came with (I think?) nwb, not the backstop code in react/backstop